host/3.0/publish-appservice-stage-sovereign-clouds.yml

```diff
@@ -145,7 +145,7 @@ steps:
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice-$(CloudName)-stage${{stage}}
           docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice-$(CloudName)-stage${{stage}}
 
-          docker push $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName-stage${{stage}}
+          docker push $TARGET_REGISTRY/python:$(TargetVersion)-appservice-$(CloudName)-stage${{stage}}

```

Comment:
what is the best way to test these changes before merging?

Reply:
Unfortunately there is no best way to test them without running the pipeline atleast once at the time of a release.

Reply:
Can we trigger release pipeline pointing to your branch?